### DISCUSSION TITLE
Wrong value selection in the options picker.

### DISCUSSION THREAD

**Discussion by cyber-kivi** [Budibase/budibase#7174] [Help] [answered]
Hello! I have a problem with option picker
There are 2 tables
1) main table has many columns and one is "status"
2) another table has 3 values
![image](https://user-images.githubusercontent.com/106427850/183644060-7eaa9e84-5089-460f-8750-80c8f7d9767e.png)
![image](https://user-images.githubusercontent.com/106427850/183644162-40c88f86-7edc-43eb-81ca-a786d2cc76a8.png)
![image](https://user-images.githubusercontent.com/106427850/183644244-3294a497-b7ae-4a44-b9be-befb9fcb3820.png)

The problem is the wrong value selection for the "status" field in the options picker.
https://user-images.githubusercontent.com/106427850/183643362-ad0728e8-37d4-4beb-b074-3573a0d232cb.mp4

**Comment by melohagan** [comment, marked as answer]
Hello @cyber-kivi 

Because you are using a Repeater, the Option Picker field names are not unique. Any fields with the same name will be tied to the same value (synced). 
There is an open issue relating to this: https://github.com/Budibase/budibase/issues/5658#issuecomment-1135881534

What might work is to repeat the entire form - so move the *New Form* inside the *Repeater*

**Reply by cyber-kivi** [reply]
Thank you! Moving the New Form inside the Repeater fixed the bug.
